.github/workflows/ci.yml

```diff
@@ -87,19 +87,20 @@ jobs:
         run: pip install maturin poetry toml
       - name: Create an isolated example directory
         run: cp -r examples/simple-extension/ ../simple-extension-msrv
-      - name: Edit Cargo.toml
+      - name: Edit Cargo.toml and change the path of rust-numpy
         run: |
           import toml
           cargo_toml = toml.load("Cargo.toml")
+          cargo_toml["dependencies"]["numpy"]["path"] = "../rust-numpy"
           cargo_toml["dependencies"]["ndarray"] = "0.13.1"
           cargo_toml["dependencies"]["num-complex"] = "0.2.4"
-          cargo_toml["dependencies"]["numpy"]["path"] = "../rust-numpy"
           with open("Cargo.toml", "w") as f:
             toml.dump(cargo_toml, f)
         working-directory: ../simple-extension-msrv
         shell: python
-      - name: Remove ndarray 0.14.0
-        run: cargo update -p ndarray:0.14.0 --precise 0.13.1
+      - name: Use ndarray 0.13.1 and num-complex 0.2.0
+        run: |
+          cargo update -p ndarray:0.15.2 --precise 0.13.1
```

Comment:
```suggestion
          cargo update -p ndarray:0.15.3 --precise 0.13.1
```

Reply:
Maybe we should checkin `Cargo.lock`

Reply:
Thanks, I came up with a complex shell hack